### ISSUE TITLE
browserslist without config

### ISSUE THREAD

**Issue by amree** [browserslist/browserslist#358]
Based on the [README](https://github.com/browserslist/browserslist#queries)

>  If the above methods did not produce a valid result Browserslist will use defaults: > 0.5%, last 2 versions, Firefox ESR, not dead.

So, if I didn't specify any config at all, should `npx browserslist` output the result based on the default config? For now, it's giving:

```
browserslist: Browserslist config was not found. Define queries or config path. 
```

Is this a bug or something that's expected?

**Comment by ai** [comment]
Fixed af3b108

**Comment by ai** [comment]
Released in 4.5.2.
